.gitignore

```diff
@@ -0,0 +1,8 @@
+/lib/helpers/secrets.py
+/.idea/.gitignore
+/.idea/apy.iml
+/.idea/codeStyles/codeStyleConfig.xml
+/.idea/misc.xml
+/.idea/modules.xml
+/.idea/inspectionProfiles/profiles_settings.xml
+/.idea/vcs.xml
```

Comment:
```suggestion
/.idea/
```

Reply:
Nice find! I ignored the files through the IDE and didn't check how it went to the .gitignore. Thank you!